Cargo.toml

```diff
@@ -1,11 +1,11 @@
 [workspace]
 resolver = "2"
 members = [
-    "engine-api",
     "evm-ext",
     "genesis",
     "genesis-builder",
     "genesis-image",
+    "json-rpc-api",
```

Comment:
Being a minimalist, I suggest to call the folder name just `api` and crate name `moved-api`.